data/json/mapgen/hospital/hospital.json

```diff
@@ -0,0 +1,115 @@
+        "#...............+..................................b#``#BB...+...+...BB#",
+        "#...............#..................................b#``#....l#...#l....#",
+        "#################F........xCCCCCCCCCx..............T#``#######...#######",
+        "#...............#.........x.c.c.c.c.x..............b#``#....l#...#l....#",
+        "#...............+........./.........x..............b#``#BB...+...+...BB#",
+        "#+#+#+#+#+#.....+........./.........x..............b#``#BB...+...+...BB#",
+        "#.#.#.#.#.#.....#.........x.........x..............b#``#...cT#...#Tc...#",
+        "#&#&#&#&#&#sssss#.D.....F.#rr###+####.DF.TbbbbTbbbbT#``#######///#######",
+        "####################//########r....l#################``#l.rrr#...#rrr.l#",
+        "#...ll...x...ll...x....#SSSTf#r...cd#s.&#&.s#s.&#&.s#``#l....!...!....l#",
+        "#.TB..BT.x.TB..BT.x....#.....+.....d#...#...#...#...#``#l.rrr#...#rrr.l#",
+        "#r.B..B.rxr.B..B.rx....#########!#####+###+###+###+##;;#######...#######",
+        "#r......rxr......rx....................................................#",
+        "#xxx//xxxxxxx//xxxx....................................................;",
+        "#......................................................................#",
+        "#......................####################...#######;;#######...#######",
+        "#......................x............#l.rrr#...#rrr.l#``#l.rrr#...#rrr.l#",
+        "#....................../............#l....!...!....l#``#l....!...!....l#",
+        "#....................../............#l.rrr#...#rrr.l#``#l.rrr#...#rrr.l#",
+        "#......................x............#######///#######``#######///#######",
+        "####################///##############...cT#...#Tc...#``#...cT#...#Tc...#",
+        "###########.......l#...x...........l#BB...+...+...BB#``#BB...+...+...BB#",
+        "###########r.TBBT..#.../.....TBBT...#BB...+...+...BB#``#BB...+...+...BB#",
+        "###########r..BB...#.../......BB...r#....l#...#l....#``#....l#...#l....#",
+        "###########........#...x...........r#######...#######``#######...#######",
+        "###########xxx//xxx#...##############....l#...#l....#``#....l#...#l....#",
+        "#l........x............x...........r#BB...+...+...BB#``#BB...+...+...BB#",
+        "#r.TBB7.../............/.....6.....r#BB...+...+...BB#``#BB...+...+...BB#",
+        "#r..BB..../............/.....*.....r#...cT#...#Tc...#``#...cT#...#Tc...#",
+        "#.........x............x...........r#######...#######``#######...#######",
+        "###########xxx//xxx#...##############...cT#...#Tc...#``#...cT#...#Tc...#",
+        "###########........#...x...........r#BB...+...+...BB#``#BB...+...+...BB#",
+        "###########r..BB...#.../......BB...r#BB...+...+...BB#``#BB...+...+...BB#",
+        "###########r.TBBT..#.../.....TBBT...#....l#...#l....#``#....l#...#l....#",
+        "###########.......l#...x...........l########...#######``#######...#######",
```

Comment:
Looks like the lines in question are number 61 and 62 here.

Reply:
Thanks! I've fixed that.